docs/framework/install/on-windows-vista.md

```diff
@@ -10,7 +10,7 @@ The .NET Framework is required to run many applications on Windows. You can use
 
 ![This application could not be started](./media/this-application-could-not-be-started.png)
 
-These instructions will help you install the .NET Framework versions you need. The [.NET Framework 4.8](https://github.com/Microsoft/dotnet/tree/master/releases/net48) is the latest version. It is not supported on Windows Vista and Windows Server 2008. It is included with [Windows 10 May 2019 Update](https://support.microsoft.com/help/4028685/windows-10-get-the-update).
+These instructions will help you install the .NET Framework versions you need. The [.NET Framework 4.8](https://github.com/Microsoft/dotnet/tree/master/releases/net48) is the latest version. It is not supported on Windows Vista and Windows Server 2008. It is included with [Windows 10 May 2019 Update](https://support.microsoft.com/help/4028685/windows-10-get-the-update) and later versions.
```

Comment:
The link should probably be on "later versions" since the target page now is for October.